src/sumoLogger.js

```diff
@@ -64,6 +67,12 @@ class SumoLogger {
         if (newConfig.returnPromise) {
             this.config.returnPromise = newConfig.returnPromise;
         }
+        if (newConfig.useIntervalOnly) {
+            this.config.useIntervalOnly;
```

Comment:
Should this be `this.config.useInterval = newConfig.useIntervalOnly` or have I missed something?

Reply:
Yes, you're correct. 😄